### DISCUSSION TITLE
showing description for checkbox selection

### DISCUSSION THREAD

**Discussion by redrobotdev** [SBoudrias/Inquirer.js#1512] [Ideas]
similar to the select method, it would be useful to show description for checkboxes when the selection is hovering over them.

**Comment by SBoudrias** [comment]
I think it does make sense to provide the API for this. Any chance you'd be interested in sending a PR?

The code is over here if you want to take a look https://github.com/SBoudrias/Inquirer.js/blob/main/packages/checkbox/src/index.mts

**Reply by SBoudrias** [reply]
Implemented https://github.com/SBoudrias/Inquirer.js/commit/e17b0c79b7e4d70ed1471eeed1433fb0837c1e94
